docs/IMAGEUPLOADER.md

```diff
@@ -45,3 +45,13 @@ Replace `XXXXXXXXXXXXXXX` with your API key from s-ul.eu. It can be found on [yo
 |Extra headers||
 |Image link|`{url}`|
 |Deletion link|`https://s-ul.eu/delete.php?file={filename}&key=XXXXXXXXXXXXXXX`|
+
+### [i.fourtf.com](https://github.com/fourtf/i)
+
```

Comment:
I think you could add some kind of information explaining that this is an example config for websites/servers hosting this project, not an actual uploading service.